library/init/meta/interactive.lean

```diff
@@ -6,8 +6,11 @@ Authors: Leonardo de Moura
 prelude
 import init.meta.tactic init.meta.rewrite_tactic init.meta.simp_tactic
 import init.meta.smt.congruence_closure init.category.combinators
+import init.meta.lean_parser
 
 namespace interactive
+@[reducible] meta def parse {α : Type} (p : lean_parser α) : Type := α
```

Comment:
I think it is feasible to build something that would work for the types we currently use at `tactic.interactive`. In general, the problem can be quite complex. This translator would have to recognize all `vm_extension`s we have. In general, `vm_obj`s might have nested closures. Some of these closures refer to auxiliary functions that only exist in the `vm`. 

Recall that we have the type `itactic` which is sugar for `tactic unit`.